keras/layers/advanced_activations.py

```diff
@@ -36,3 +37,41 @@ def get_output(self, train):
     def get_config(self):
         return {"name":self.__class__.__name__,
         "input_shape":self.input_shape}
+
+
+class TLinear(Layer):
```

Comment:
There's no reason to not make these both extend `MaskedLayer` instead of `Layer` and thereby also be supported in masked scenarios.